### PR TITLE
Node installation

### PR DESCRIPTION
**Pull request by jhonhquintero**

### CODE REVIEW

File: node_modules_locally.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+1. Create this file: ~/.npmrc
+2. Add this line to the file: 
+- prefix = ${HOME}/.npm-packages
+3. Create npm packages folder:
+- mkdir ~/.npm-packages`
+4. Add this line to .bashrc:
+- PATH=$PATH:$HOME/.npm-packages/bin # Ensure user-global npm binaries can execute.`
+
```
Comment (alboyadjian) [inline node_modules_locally.md:8]: @eknelson17  Could you please add details of how you got Sublime configured to work with the local copy?

**Comment by dreampopclub** [comment]
This looks good to me. Merge?

File: node_modules_locally.md (hunk 1 above)
Comment (alboyadjian) [inline node_modules_locally.md:8]: @mr-sudaca  Please also add instrux for installing the eslint-config-airbnb module from OUR repo vs from official repos.  Explain the benefits of doing this, and these steps (w/ some extra explanation)

```
npm -g install eslint
git clone git@github.com:vhl/javascript-style-guide
cd javascript-style-guide/packages/eslint-config-airbnb
npm -g install
```

Comment (eknelson17) [inline node_modules_locally.md:8]: @vhladama I added the process that finally worked.

**Pull request merged**